projects/sturdy/index.js

```diff
@@ -24,27 +31,37 @@ const priceInUSD = (value, decimals, price) => {
     return (value / Math.pow(10, decimals) * price / Math.pow(10, 8)).toFixed(2);
 }
 
-async function fetch(borrow) {
-    const { reserves } = await request(graphUrl, graphQuery);
-    const tvl = reserves.reduce((sum, reserve) => {
-        const value = borrow ? reserve.availableLiquidity : reserve.totalLiquidity;
+async function fetch(borrow, chain) {
+    const graphUrl = chain == 'fantom' ? graphUrl_ftm : graphUrl_eth;
+    const { reserves, usdPriceEth } = await request(graphUrl, graphQuery);
+    let tvl = reserves.reduce((sum, reserve) => {
+        const value = borrow ? reserve.totalCurrentVariableDebt : reserve.totalLiquidity;
```

Comment:
again, shouldnt it be `reserve.availableLiquidity` instead of `reserve.totalLiquidiy` to get tvl that is not borrowed?

Reply:
[reserve.totalLiquidiy] should be used for TVL.

Reply:
No, doesn't include the amount lent

Reply:
@mingji I believe `availableLiquidity` would be correct here as well

Reply:
Thanks for your support. I fixed for it.